R/utilities.R

```diff
@@ -3048,29 +3163,120 @@ get_bams = function(sample,
 #' # then launch IGV (e.e. from a conda installation):
 #' # conda activate igv; igv &
 #' # this_sv = annotated_sv %>% filter(gene=="ETV6")
-#' # tumour_bam = get_bams(sample=this_sv$tumour_sample_id)
+#' # tumour_bam = get_bams(sample=this_sv$tumour_sample_id) #you don't need to know the details for the bam file but you can supply it if you want
+#' # socket = make_igv_snapshot() #run with no arguments to get the socket for a running IGV instance
 #' # make_igv_snapshot(chrom=this_sv$chrom2, start=this_sv$start2, end=this_sv$end2, this_sample_id=this_sv$tumour_sample_id,out_path="~/IGV_snapshots/")
-#'
+#' # this_mutation = get_coding_ssm(seq_type="capture") %>% head(1)
+#' # make_igv_snapshot(socket=socket,sample_ids=this_mutation$Tumor_Sample_Barcode,this_seq_type="capture",colour_by="READ_STRAND")
+#' # run on a bunch of variants using apply:
+#' # apply(to_snapshot,1,function(x){make_igv_snapshot(sample_ids=x["sample_id"],
+#'       seq_type_filter="capture",chrom=x["chr"],start=x["start"],end=x["end"],
+#'       details=paste0(x["ref"],"-",x["alt"]),
+#'       gene=x["Hugo_Symbol"],socket=socket)})
+#'       
 make_igv_snapshot = function(bams,
+                             sample_ids,
+                             this_seq_type="genome",
```

Comment:
Update the parameter descriptions to match the new/deprecated parameters of this function.

Reply:
I hope most are self-explanatory. I tried to fix the ones with issues.